packages/fxa-admin-server/src/gql/account/account.resolver.spec.ts

```diff
@@ -192,34 +194,36 @@ describe('AccountResolver', () => {
     )) as Account;
     expect(updatedResult).toBeDefined();
     expect(updatedResult.disabledAt).toBeNull();
-    expect(logger.info).toBeCalledTimes(1);
+    expect(logger.info).toBeCalledTimes(2);
   });
 
   it('does not locate non-existent users by uid', async () => {
     const result = await resolver.accountByUid(USER_2.uid, 'joe');
     expect(result).toBeUndefined();
-    expect(logger.info).toBeCalledTimes(1);
+    expect(logger.info).toBeCalledTimes(2);
   });
 
   it('locates the user by email', async () => {
     const result = (await resolver.accountByEmail(
       USER_1.email,
+      true,
```

Comment:
Q: What does the new third Boolean arg mean?
Ah... I think `autoCompleted`, per [packages/fxa-admin-server/src/gql/account/account.resolver.ts#L114](https://github.com/mozilla/fxa/pull/13742/files#diff-0558e3bc348d6e8f9a979bed76128d9beeb525124beeb71e34ee2bdfa7fd8723R114).

```ts
@Args('autoCompleted', { nullable: false }) autoCompleted: boolean,
```